block.go

```diff
@@ -273,6 +273,128 @@ func (b ChildPageBlock) GetType() BlockType {
 	return b.Type
 }
 
+type EmbedBlock struct {
+	Object         ObjectType `json:"object"`
+	ID             BlockID    `json:"id,omitempty"`
+	Type           BlockType  `json:"type"`
+	CreatedTime    *time.Time `json:"created_time,omitempty"`
+	LastEditedTime *time.Time `json:"last_edited_time,omitempty"`
+	HasChildren    bool       `json:"has_children,omitempty"`
+	Embed          Embed      `json:"embed,omitempty"`
+}
+
+func (b EmbedBlock) GetType() BlockType {
+	return b.Type
+}
+
+type Embed struct {
+	Caption []RichText `json:"caption,omitempty"`
+	URL     string     `json:"url"`
+}
+
+type ImageBlock struct {
+	Object         ObjectType `json:"object"`
+	ID             BlockID    `json:"id,omitempty"`
+	Type           BlockType  `json:"type"`
+	CreatedTime    *time.Time `json:"created_time,omitempty"`
+	LastEditedTime *time.Time `json:"last_edited_time,omitempty"`
+	HasChildren    bool       `json:"has_children,omitempty"`
+	Image          Image      `json:"image,omitempty"`
```

Comment:
Why `omitempty` is used here? As I understood, this prop can't be empty.

Reply:
Yep. I got that fixed across all the structs as well. Thanks